### ISSUE TITLE
Something looks wrong with node hostnames/droplet names on digitalocean

### ISSUE THREAD

**Issue by looztra** [kris-nova/kubicorn#143]
```
kubicorn version: master at 28dcd2c
env: fedora 26 with kernel 4.11.11-300.fc26.x86_64
```
I created a cluster with 1 master and 3 nodes on digitalocean.

```
Cloud: digitalocean
KubernetesApi:
  Endpoint: <redacted>
  Port: "443"
  metadata:
    creationTimestamp: null
Location: Amsterdam 2
Name: dok8s
Network:
  CIDR: ""
  Identifier: ""
  Type: ""
  metadata:
    creationTimestamp: null
ServerPools:
- BootstrapScript: ""
  Firewalls: null
  Identifier: ""
  Image: ubuntu-16-04-x64
  MaxCount: 1
  MinCount: 0
  Name: dok8s-master
  Size: 1gb
  Subnets: null
  Type: master
  metadata:
    creationTimestamp: null
- BootstrapScript: ""
  Firewalls: null
  Identifier: ""
  Image: ubuntu-16-04-x64
  MaxCount: 3
  MinCount: 0
  Name: dok8s-node
  Size: 1gb
  Subnets: null
  Type: node
  metadata:
    creationTimestamp: null
```

Provisionning went well:

```
2017-08-04T14:46:41+02:00 [✿]  Wrote kubeconfig to [/home/cfurmaniak/.kube/config]
2017-08-04T14:46:41+02:00 [✿]  The [dok8s] cluster has applied successfully!
2017-08-04T14:46:41+02:00 [✿]  You can now `kubectl get nodes`
2017-08-04T14:46:41+02:00 [✿]  You can SSH into your cluster ssh -i ~/.ssh/id_rsa root@xx.yy.zz.ww
```

But the cluster only shows a single node:

```
└> kubectl get nodes --show-labels
NAME           STATUS    AGE       VERSION   LABELS
dok8s-master   Ready     24m       v1.7.3    beta.kubernetes.io/arch=amd64,beta.kubernetes.io/os=linux,kubernetes.io/hostname=dok8s-master,node-role.kubernetes.io/master=
dok8s-node     Ready     20m       v1.7.3    beta.kubernetes.io/arch=amd64,beta.kubernetes.io/os=linux,kubernetes.io/hostname=dok8s-node
```

However I can see the 3 nodes in the digitalocean console, all named `dok8s-node`

When I try to delete the cluster:
```
> kubicorn delete -n dok8s
2017-08-04T14:57:53+02:00 [✔]  Selected [fs] state store
2017-08-04T14:57:55+02:00 [✖]  Unable to destroy resources for cluster [dok8s]: Found [3] Droplets for Name [dok8s-node]
```

My first guess is that if k8s only sees one node it's because they all have the same hostname (which is by default related to the droplet name in digitalocean)

**Comment by k4k** [comment]
@looztra what happens if you set `MinCount: 3` instead of `MaxCount: 3` on the node definition? As you have it defined there, the output you are seeing seems correct as you're defining a ceiling and not a floor.

**Comment by kris-nova** [comment]
@looztra can you share the logs in `/var/log/meshbird.log` and both `/var/log/cloud-init.log` and `/var/log/cloud-init-output.log`?

I know it will take a minute or two for the cluster to come up and all the nodes to show ready, but this seems like something else went wrong.

**Comment by kris-nova** [comment]
@looztra I think your hostname assessment is right pushing up a patch now

**Comment by kris-nova** [comment]
Patch is released in #144 can you please try again - sorry about that :disappointed:

**Comment by looztra** [comment]
It's ok now indeed:
```
└> kubectl get nodes
NAME             STATUS    AGE       VERSION
dok8s-master-0   Ready     1h        v1.7.3
dok8s-node-0     Ready     1h        v1.7.3
dok8s-node-1     Ready     1h        v1.7.3
dok8s-node-2     Ready     1h        v1.7.3

```

thx!
